lib/Driver/DarwinToolChains.cpp

```diff
@@ -585,9 +585,45 @@ bool toolchains::Darwin::shouldStoreInvocationInDebugInfo() const {
   return false;
 }
 
+static void validateDeploymentTarget(const toolchains::Darwin& TC,
```

Comment:
Nit: We prefer the space before the `&`, not after.

Reply:
Sure, fixed :))